### PR TITLE
KAFKA-16467: Add how to integrate with kafka repo

### PR DESCRIPTION
**Pull request by FrankYang0529**
ref: https://issues.apache.org/jira/browse/KAFKA-16467

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,30 @@ You can run it with the following command, note that it requires docker:
 
 Then you can open [localhost:8080](http://localhost:8080) on your browser and browse the documentation.
 
-To kill the process, just type ctrl + c
\ No newline at end of file
+To kill the process, just type ctrl + c.
+
+## Preview latest kafka document
```
Comment (showuon) [inline README.md]: We could be consistent with the L1 title. Ex: 
`How to preview the latest documentation changes in Kafka repository?`

File: README.md [hunk 2]
```diff
@@ -10,4 +10,30 @@ You can run it with the following command, note that it requires docker:
...
+
+1. Assume you have [kafka](https://github.com/apache/kafka) and kafka-site folder structure like this:
+
+```shell
+.
+├── kafka
+└── kafka-site
+```
+
```
Comment (showuon) [inline README.md]: I think we don't need to have this assumption.

File: README.md [hunk 3]
```diff
@@ -10,4 +10,30 @@ You can run it with the following command, note that it requires docker:
...
+2. Generate document in kafka folder:
```
Comment (showuon) [inline README.md]: Generate document in kafka folder: -> Generating document from kafka repository:

File: README.md [hunk 4]
```diff
@@ -10,4 +10,30 @@ You can run it with the following command, note that it requires docker:
...
+```
+
+3. Running website in kafka-site folder and open [http://localhost:8080/dev/documentation/](http://localhost:8080/dev/documentation/):
+
+```shell
+rm -rf dev
+mkdir dev
+cp -r ../kafka/site-docs/* dev
+./start-preview.sh
+```
```
Comment (showuon) [inline README.md]: How about this:

3. Copying the generated documents in Kafka repository into kafka-site, and preview them (note that it requires docker):

```shell
# change directory into kafka-site repository
cd KAFKA_SITE_REPO
# copy the generated documents into dev folder
rm -rf dev
mkdir dev
cp -r ../kafka/site-docs/* dev
# preview it
./start-preview.sh
```

Then you can open [http://localhost:8080/dev/documentation/](http://localhost:8080/dev/documentation/) on your browser and browse the generated documentation.

File: README.md [hunk 5]
```diff
@@ -10,4 +10,30 @@ You can run it with the following command, note that it requires docker:
...
+├── kafka
+└── kafka-site
+```
+
+2. Generate document in kafka folder:
+
+```shell
+./gradlew clean siteDocTar
+tar zxvf core/build/distributions/kafka_2.13-$(./gradlew properties | grep version: | awk '{print $NF}' | head -n 1)-site-docs.tgz
+```
```
Comment (showuon) [inline README.md]: We didn't mention we are in kafka repo now. How about this:


```shell
# change directory into kafka repository
cd KAFKA_REPO
./gradlew clean siteDocTar
# supposing built with scala 2.13
tar zxvf core/build/distributions/kafka_2.13-$(./gradlew properties | grep version: | awk '{print $NF}' | head -n 1)-site-docs.tgz
```

**Review by showuon — COMMENTED** [review]
@FrankYang0529 , thanks for the PR. Left some comments.

**Comment by FrankYang0529** [comment]
Hi @showuon, thanks for reviewing. I've addressed all comments and add you as co-author.

File: README.md [hunk 6]
```diff
@@ -10,4 +10,32 @@ You can run it with the following command, note that it requires docker:
...
+2. Copying the generated documents from Kafka repository into kafka-site, and preview them (note that it requires docker):
+
+```shell
+# change directory into kafka-site repository
+cd KAFKA_SITE_REPO
+# copy the generated documents into dev folder
+rm -rf dev
+mkdir dev
+# change directory into kafka repository
+cp -r KAFKA_REPO/site-docs/* dev
```
Comment (showuon) [inline README.md]: I don't think this comment is correct.

Comment (FrankYang0529) [reply README.md]: I remove `# change directory into kafka repository`. Thank you.

**Review by showuon — APPROVED** [review]
LGTM!

**Pull request merged**